### PR TITLE
Add enterprise logo and contact name to DFC API with standard attributes

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

Some enterprise data was added as custom attributes because the DFC didn't support them yet. But now we have a standard and should use it. I'm keeping the old fields for backwards-compatibility. We may remove them in the future.

#### What should we test?

- No test needed.

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [x] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: engines/dfc_provider/app/services/enterprise_builder.rb [hunk 1]
```diff
@@ -60,4 +61,14 @@ def self.enterprise_group(group)
...
+
+  def self.contact(enterprise)
+    firstName, lastName = enterprise.contact_name&.split(/ ([^ ]+)$/) # rubocop:disable Naming/VariableName
+
+    DataFoodConsortium::Connector::Person.new(
+      urls.enterprise_url(enterprise.id, anchor: "mainContact"),
+      firstName:, # rubocop:disable Naming/VariableName
+      lastName:, # rubocop:disable Naming/VariableName
+    )
+  end
```
Comment (rioug) [inline engines/dfc_provider/app/services/enterprise_builder.rb:73]: I am tempted to request a spec for this, but it seems trivial enough.

**Review by rioug — APPROVED** [review]
Good to be moving away from the custom attributes !

File: swagger/dfc.yaml [hunk 2]
```diff
@@ -99,7 +99,6 @@ paths:
                     - "@id": http://test.host/api/dfc/enterprises/10000
                       "@type": dfc-b:Enterprise
                       dfc-b:hasAddress: http://test.host/api/dfc/addresses/40000
-                      dfc-b:logo: ''
```
[the remark this answers is not in the document]
Comment (mkllnk) [reply swagger/dfc.yaml:102]: Yes, it does initialise all attributes with blank values but that will change in the next release. My pull request was init with nil values was merged.

**Comment by mkllnk** [comment]
> I had a quick look in the codebase and found that we once had to split first and lastnames in the database

Yes, but the reasoning is not very clear on the related issue.
* #8577

It mentions reports etc but that may just be because Spree distinguished between first and last name. So we needed consistency. But the issue is also linked to the first DFC work and the DFC has firstName and lastName on a Person. I think that it's unfortunate that we have to split the name. Many regions handle names differently and it's not feasible for us to handle all of those possibilities. Having one full name is much more inclusive and easier in the UI. One side effect is that you can't fake friendliness in emails saying "Hi {first name}!". :wink:

**Pull request merged**